crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -610,7 +650,9 @@ async fn enqueue_block(
 }
 
 #[instrument(skip_all, err)]
-async fn get_sequencer_chain_id(client: SequencerClient) -> eyre::Result<tendermint::chain::Id> {
+pub(crate) async fn get_sequencer_chain_id(
```

Comment:
Please don't make this crate public here. I'd rather have this method copied to the `sequencer` module.